run.sh

```diff
@@ -8,6 +8,18 @@
 # which causes the password to be incorrect.
 # To test that try running `docker run --rm --env-file .env busybox /bin/sh -c 'echo $NODE_ECDSA_KEY_PASSWORD'`
 # This will output password with single quote. Not sure why this happens.
+# Function to read Docker secrets
+read_secret() {
```

Comment:
where is this used?

Reply:
It's used to read the content of Docker secrets (ecdsa_key_password and bls_key_password) and pass them to the Docker containers securely.
--volume "ecdsa_key_password:/run/secrets/ecdsa_key_password:ro" \
--volume "bls_key_password:/run/secrets/bls_key_password:ro" \
These lines in the docker run command mount the Docker secrets into the appropriate paths within the containers. The "read_secret()" function ensures that the contents of these secrets are correctly provided when the script is executed.

Reply:
ok thanks. I will test this and update. thanks for the PR. appreciate it.

Reply:
> ok thanks. I will test this and update. thanks for the PR. appreciate it.

hey, Is it passed the test?